### ISSUE TITLE
Image Search returns un-safe/unfiltered images from Pixabay.com

### ISSUE THREAD

**Issue by jmg460** [photopea/photopea#2604]
When testing the remote image search feature, the Gallery can include un-safe images from pixabay.com.  These images are not appropriate for children in the United States.  Solution: set the 'safesearch' flag to 'true' on the pixabay.com API: https://pixabay.com/api/docs/

**Comment by photopea** [comment]
Hi, thanks for your report. On the other hand, some users of Photopea might want to find and use precisely these "unsafe" images. How do we find out, if the visitor is an adult or not? What if we added a checkbox "safe images only"?

You can ask your school not to use Photopea, until we solve it somehow.

**Comment by jmg460** [comment]
>>some users of Photopea might want to find and use precisely these "unsafe" images<< I totally understand this. But, I think customizing Photopea for student use will work out favorably for you as students that learn your tool will use it as adults... This will lead to more ad revenue.  

As a teacher, I would like to continue the use of Photopea.com because it is an **_excellent_** alternative to Photoshop.  I solved the "unsafe" issue already.  (We blocked all resources that come from Pixabay.com). So, my issue report on github is really just to help other teachers and keep other students safe.  Personally, I don't think a checkbox will deter children from searching.  And, adults searching for unsafe images probably know where to find them.  Additionally, if you open up pixabay.com, by default, their safe search is on until you turn it off.  That was my thinking... But, it is only one opinion.  It's also worth mentioning that I am a very satisfied web 'customer' with the exception of this one tiny issue.

Bottom line, I reached out because Photopea is fantastic and I want to continue using it.

**Comment by photopea** [comment]
I guess you are right. I enabled "safe-search" for all searches. If someone is looking for photos of woman in bikini, they would have to find it elsewhere.

BTW. the Gallery Plugin is not a standard part of Photopea. It is an independent website: https://www.photopea.com/plugins/gallery.html , which can be inserted into other websites, and send messages to a parent website (like "open a photo from this URL").

You can create your own plugins for your school, it is very simple :) https://www.photopea.com/api/plugins

**Comment by jmg460** [comment]
Wow, that was quick!  You are awesome!

To all K-12 technology or art teachers, if you are searching for a free alternative to Adobe Photoshop to use on Chromebooks, try [Photopea.com](Photopea.com).  I highly recommend this photo manipulation and image editing tool!  [Photopea.com](Photopea.com) is light years beyond any other web-based photo editor.
